lib/generators/module/USAGE

```diff
@@ -1,23 +1,23 @@
 Description:
-    Creates a Rails engine starter app that wires up a service backed endpoint.
+    This generator will create an initial file structure and engine file for VFS developers.  The purpose of this being so they can quickly start developing code in an isolated file structure exclusive to the new business logic thay are creating.
+
+When a generator is invoked, each public method in the generator is executed sequentially in the order that it is defined.
 
 Example:
-    rails va new foo
+    rails generate module Thing
```

Comment:
i don't love the name "Thing" for the example, but I never liked "foo" or "HelloWorld" either.  Personally I'd pick something like "CoolNewVAFeature" but I'm really just bikeshedding here.  Feel free to ignore this, lol